### PR TITLE
Fix Summary test

### PR DESCRIPTION
**Pull request by dianzrong**
**What kind of change does this PR introduce?**

Bug fix

**Summary**

Make sure "punkt" from NLTK is downloaded

**Checklist**
* [x] My code follows the style guidelines of OpenAdapt
* [x] I have perfomed a self-review of my code
* [x] If applicable, I have added tests to prove my fix is functional/effective
* [x] I have linted my code locally prior to submission
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation (e.g. README.md, requirements.txt)
* [x] New and existing unit tests pass locally with my changes

**How can your code be run and tested?**

It can be run with pytest

### CODE REVIEW

File: openadapt/strategies/mixins/summary.py [hunk 1]
```diff
@@ -10,12 +10,13 @@ class MyReplayStrategy(SummaryReplayStrategyMixin):
 from openadapt.models import Recording
 from openadapt.strategies.base import BaseReplayStrategy
 
-
 from sumy.summarizers.lsa import LsaSummarizer
 from sumy.parsers.plaintext import PlaintextParser
 from sumy.nlp.tokenizers import Tokenizer
 from sumy.nlp.stemmers import Stemmer
 from sumy.utils import get_stop_words
+import nltk
```
Comment (KrishPatel13) [inline openadapt/strategies/mixins/summary.py:18]:
```
from loguru import logger
from sumy.nlp.stemmers import Stemmer
from sumy.nlp.tokenizers import Tokenizer
from sumy.parsers.plaintext import PlaintextParser
from sumy.summarizers.lsa import LsaSummarizer
from sumy.utils import get_stop_words
import nltk

from openadapt.models import Recording
from openadapt.strategies.base import BaseReplayStrategy
```

- Standard Imports followed by 3rd Party Imports followed by Local imports.
1 newline separation will be great between them. :)

- When it comes to ordering import ABC vs from ABC import XYZ. It is more readable to follow one specific format for all scripts/project. 

- When it comes to OpenAdapt. I have observed that we are following to  put from *** before import ***.
   See: https://github.com/MLDSAI/OpenAdapt/blob/main/openadapt/record.py#L18
  eg.
  ```
  from ABC import XYZ
  import ABC
  ```

- When it comes to order two similar imports  we are following alphabetical order: eg.
```
from A import W
from A import X
```

**Review by KrishPatel13 — CHANGES_REQUESTED** [review]

File: openadapt/strategies/mixins/summary.py (hunk 1 above)
Comment (dianzrong) [reply openadapt/strategies/mixins/summary.py:16]: Good point!! Addressed in [f2671f7](https://github.com/MLDSAI/OpenAdapt/pull/306/commits/f2671f7ba6715e5874fbec986ba9d01d62ba4d0e)

**Review by KrishPatel13 — APPROVED** [review]
Approved!

**Review by KrishPatel13 — APPROVED** [review]
Reviewed, Approved!

**Pull request merged**